contracts/FlashMinter.sol

```diff
@@ -5,6 +5,7 @@ interface FlashMinterLike {
     function balanceOf(address) external returns (uint256);
     function executeOnFlashMint(uint256 value, bytes calldata data) external;
     function flashMint(uint256, bytes calldata) external;
+    function flashReturn(uint256 value) external;
```

Comment:
Given that the flashMint is resolved within the same transaction, what would be the use of simultaneous flashMints?

As I see it that would only allow the same user, during the same transaction, to call `flashMint` more than once, something that is possible to do safely in the original code just by removing the lock.

Reply:
I'm not sure there's any good reason to actually do simultaneous flashMints, it's more that this seemed like a more elegant implementation of flashMinting to me, and it removes some restrictions (whether useful or not)